taiga_halo2/src/transaction.rs

```diff
@@ -336,8 +349,9 @@ pub mod testing {
         (bundle, r_vec)
     }
 
+    #[cfg(feature = "borsh")]
     #[test]
-    fn test_halo2_transaction() {
+    fn test_halo2_transaction_borsh_serialize() {
```

Comment:
We probably could separate the transaction test and the borsh test as we've added the feature flag?

Reply:
Ah, I didn't realize this was a combined test. Yeah, I can separate them. CI runs tests with `--all-features` anyway, so all feature tests are run too. But yeah, let me push an updated test.

Reply:
Updated the tests.

Reply:
Is it worth using the exported TX from `test_halo2_transaction` in `test_halo2_transaction_borsh_serialize` to avoid duplicated TX construction since it takes a long time to build a TX?

Reply:
I don't know how to pass data between tests though.

Reply:
Actually, fixed that